python/cuml/test/test_fil.py

```diff
@@ -128,6 +129,53 @@ def test_fil_classification(n_rows, n_columns, num_rounds, tmp_path):
     assert fil_acc == pytest.approx(xgb_acc, 0.01)
     assert array_equal(fil_preds, xgb_preds_int)
 
+@pytest.mark.parametrize('n_rows', [ unit_param(10000)])
+@pytest.mark.parametrize('n_columns', [unit_param(20)])
+@pytest.mark.parametrize('n_estimators', [unit_param(10)])
```

Comment:
we'll probably want to add some more params over time including 1 estimator (check degenerate case)
Probably try a very shallow max_depth and a very deep one too.

Reply:
Done.